src/Tickets/Commerce/Order_Modifiers/Modifier_Admin_Handler.php

```diff
@@ -333,7 +333,7 @@ public function handle_form_submission(): void {
 		}
 
 		// Get the raw POST data, and set the modifier ID.
-		$raw_data                      = tribe_get_request_vars();
+		$raw_data                      = tribe_get_request_vars( true );
```

Comment:
I set this to true, because it was causing failures in the tests. `true` disables caching of the request_vars.

Reply:
Thats good. Either use it with true or even better use tec_get_request_var or tec_get_request_var_raw so we dont grab the whole _REQUEST, but specifics.

Reply:
Good point, since it was an easy change I moved towards `tec_get_request_var`.

Although, that brings out a question, is `tribe_get_request_vars` the correct call? The idea was that we passed all of the data to the strategy to figure out what it needed via `map_form_data_to_model`. Maybe the correct change is to remove the `$raw_data`, run `map_form_data_to_model()` and in that method we use the individual `tec_get_request_var`. What do you think?